src/components/modal/v2/parts/views/ProductList/styles.scss

```
@@ -2,6 +2,8 @@
 @import '../../../styles/globals/mixins';
 
```

```suggestion

.header__content {
    padding-bottom: 20px;

    @include mobile {
        padding-bottom: 0px;
    }

    h1 {
        font-size: 2.375rem;
    }
}

```
Small suggestion of style tweaks for when the headline is changes to "Buy now, pay over time".